<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormattedTextWidget</class>
 <widget class="QWidget" name="FormattedTextWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="m_isFormattedTextMode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Formatted text</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="m_fontFamily">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_fontSize">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>72</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>96</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolBar" name="m_formattedTextToolBar">
       <property name="windowTitle">
        <string>toolBar</string>
       </property>
       <addaction name="separator"/>
       <addaction name="m_actionBold"/>
       <addaction name="m_actionItalics"/>
       <addaction name="m_actionUnderlined"/>
       <addaction name="m_actionColor"/>
       <addaction name="m_actionAddImage"/>
       <addaction name="m_actionAddLink"/>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="m_description"/>
   </item>
  </layout>
  <action name="m_actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/format-text-bold.png</normaloff>:/icons/format-text-bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
  </action>
  <action name="m_actionItalics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/format-text-italic.png</normaloff>:/icons/format-text-italic.png</iconset>
   </property>
   <property name="text">
    <string>Italics</string>
   </property>
   <property name="toolTip">
    <string>Italics</string>
   </property>
  </action>
  <action name="m_actionUnderlined">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/format-text-underline.png</normaloff>:/icons/format-text-underline.png</iconset>
   </property>
   <property name="text">
    <string>Underlined</string>
   </property>
   <property name="toolTip">
    <string>Underlined</string>
   </property>
  </action>
  <action name="m_actionAddImage">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/insert-image.png</normaloff>:/icons/insert-image.png</iconset>
   </property>
   <property name="text">
    <string>Add image</string>
   </property>
   <property name="toolTip">
    <string>Add image</string>
   </property>
  </action>
  <action name="m_actionAddLink">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/insert-link.png</normaloff>:/icons/insert-link.png</iconset>
   </property>
   <property name="text">
    <string>Add link</string>
   </property>
   <property name="toolTip">
    <string>Add link</string>
   </property>
  </action>
  <action name="m_actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="libmarble.qrc"/>
 </resources>
 <connections/>
</ui>
